examples/influxdb-grafana/run-psutil.sh

```diff
@@ -22,20 +22,26 @@ die () {
     exit 1
 }
 
-[ "$#" -eq 1 ] || die "Error: Expected to get one or more machine names as arguments."
-[ "${SNAP_PATH}x" != "x" ] || die "Error: SNAP_PATH must be set"
-command -v docker-machine >/dev/null 2>&1 || die "Error: docker-machine is required."
-command -v docker-compose >/dev/null 2>&1 || die "Error: docker-compose is required."
-command -v docker >/dev/null 2>&1 || die "Error: docker is required."
-command -v netcat >/dev/null 2>&1 || die "Error: netcat is required."
-file $SNAP_PATH/plugin/snap-plugin-collector-psutil >/dev/null 2>&1 || die "Error: missing $SNAP_PATH/build/plugin/snap-plugin-collector-psutil"
-file $SNAP_PATH/plugin/snap-plugin-publisher-influxdb >/dev/null 2>&1 || die "Error: missing $SNAP_PATH/build/plugin/snap-plugin-publisher-influxdb"
-
-
+# add some color to the output
+red=`tput setaf 1`
+green=`tput setaf 2`
+reset=`tput sgr0`
 
-#docker machine ip
-dm_ip=$(docker-machine ip $1) || die
-echo "docker machine ip: ${dm_ip}"
+# verify deps and the env
+[ "${SNAP_PATH}x" != "x" ] || die "Error: SNAP_PATH must be set"
+type docker-compose >/dev/null 2>&1 || die "Error: docker-compose is required."
+type docker >/dev/null 2>&1 || die "Error: docker is required."
+type netcat >/dev/null 2>&1 || die "Error: netcat is required."
+if type docker-machine >/dev/null 2>&1; then
+	docker-machine active >/dev/null 2>&1 || die "no active docker host found"     
+    dm_ip=$(docker-machine ip $(docker-machine active)) || die
+	if [ "$dm_ip" = "" ]; then 
+		die "Unable to identify IP for your docker-machine make sure that it's started"
```

Comment:
Unable to identify IP for your docker-machine. Make sure that it's started.

Maybe have the Error: preface for the two errors you added to match the rest and have them be more noticeable?